sys/auto_init/saul/auto_init_io1_xplained.c

```diff
@@ -73,8 +73,9 @@ void auto_init_io1_xplained(void)
 
         /* GPIOs */
         for (unsigned j = 1; j < 4; j++) {
-            saul_entries[i * 4 + j].dev = &(io1_xplained_saul_gpios[j - 1]);
```

Comment:
Do you know, what the 4 is about, btw?

Reply:
It corresponds to the number of extension peripherals supported by the driver (temperature + led + 2 gpios)

Reply:
Mhh maybe also in the follow-up have this be a `define`? (in case the SD-card is one day supported e.g. ;-))